src/arc.ts

```diff
@@ -259,6 +265,10 @@ export class Arc {
         contractClass = require('@daostack/arc/build/contracts/ContributionReward.json')
         contract = new this.web3.eth.Contract(contractClass.abi, addresses.base.ContributionReward, opts)
         return contract
+      case 'GEN':
```

Comment:
why duplicate this code as GEN and DAOToken?

Reply:
it 's in anticipation of a change in the migration repository which deploys two separate contracts...